html-forms/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  Email
+  <form action="none">
+    <input type="text" name="WhyCoding" id="WhyIsCodingCool">
+    <label for="WhyIsCodingCool"><div>Why is coding cool?</div></label>
+    <select name="coding-cool">
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+    <div>
+      Favorite Language (Choose One)
+      <div>
+        <input type="radio" name="favLanguage" value="HTML" id="HTML"/>
+        <label for="HTML">HTML</label>
+        <input type="radio" name="favLanguage" value="CSS" id="CSS"/>
+        <label for="CSS">CSS</label>
+        <input type="radio" name="favLanguage" value="JavaScript" id="JavaScript"/>
+        <label for="JavaScript">JavaScript</label>
+      </div>
+    </div>
+    <div>
+      Would you like to sign up for more information?
+    </div>
+    <input type="checkbox" name="signup" id="comments">Sign me up!
```

Comment:
The text labelling the checkbox should be in a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!